packages/portal/src/app/shared/guards/auth-guard.service.ts

```diff
@@ -1,11 +1,11 @@
 import { Injectable } from "@angular/core";
-import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot } from "@angular/router";
+import { ActivatedRouteSnapshot, Router, RouterStateSnapshot } from "@angular/router";
 import { AuthService } from "../authentication/auth.service";
 
 @Injectable({
   providedIn: "root",
 })
-export class AuthGuard implements CanActivate {
+export class AuthGuard  {
```

Comment:
should not this be converted to a function?

Reply:
Yes. Will update

Reply:
That's one of the possibilities. Yes.

Reply:
Great. Will do that.

Reply:
Hi. I have addressed review comments.

Reply:
Hi Sumit, yes, thank you. I have been trying to open the PR in codespaces unsuccessfully. Can you please set the target branch to `feat/develop-pr219`, thank you!

Reply:
Thank you! I will merge to this branch and test.